id-repository/id-repository-identity-service/pom.xml

```diff
@@ -109,6 +114,11 @@
 			<groupId>io.mosip.kernel</groupId>
 			<version>1.2.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
```

Comment:
why you have added this dependency

Reply:
This is new dependency kernel id object validator moved from runtime to pom dependency